src/index.js

```diff
@@ -155,6 +156,15 @@ const DidomiSDK = ({
     return apiKeyProp || apiKey;
   }
 
+  /**
+   * Determine whether the TCF is enabled from the props or from the config
+   */
+  const isTCFEnabled = config && config.app
```

Comment:
@MaximePlancke I am reading from both `tcfEnabled` and `config.app.vendors.iab.enabled`.

We could also decide that `tcfEnabled` should be set into `config.app.vendors.iab.enabled` and that the prop overrides the config. I think that would make sense here.

Reply:
Yes that works for me

Reply:
Done in https://github.com/didomi/react/pull/27/commits/6e91fc3a7499ba64d696777b57221f44ba117ccb